### ISSUE TITLE
Checkbox for travel policy

### ISSUE THREAD

**Issue by jpstroop** [pulibrary/approvals#1143]
### User story
As an administrator, I need to ensure that travelers are aware of and agree to adhere to the Library's travel policy. As such, I would like a checkbox added to the form with the text "I agree that this proposal aligns with the policies described in the [Library travel policy](https://pul-confluence.atlassian.net/wiki/spaces/LSC/pages/1933321/Princeton+University+Library+Travel+Policy+and+Best+Practices) and that I may be required to reimburse the University for expenses that are outside of the figures provided here" (or something like that). 

The form should be able to be submitted without a ✅ .

**Comment by kevinreiss** [comment]
Hi @jpstroop per you last comment did you that form should NOT be able to be submitted with a selected checkbox?

**Comment by jpstroop** [comment]
Correct. It should not be possible to submit the form without checking the box
